kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/extensions/interaction-extension-classes.owl

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
```

Comment:
Could you describe in more detail what is happening in this file?  Are we actually defining a new GO term (in the 'obo' namespace)? 

I'm also trying to understand the direction of the subclassing.  Is the goal to be able to be able to do `subclassOf* GO_MI_EXT_binding_or_direct_interaction` queries?